src/Transition/InputRulesFiles/Rules24-1-0-to-24-2-0.md

```diff
@@ -24,7 +24,19 @@ The alpha/numeric A/N numbers are not as important here.
 
 # Object Change: ObjectStartsWithG
 
-# Object Change: ObjectStartsWithH
+# Object Change: HeatPump:PlantLoop:EIR:Cooling
+
+Insert new field A7 which is also F7, "Heat Recovery Inlet Node Name", defaulted to blank?
+Insert new field A8 which is also F8, "Heat Recovery Outlet Node Name", defaulted to blank?
+Insert new field N3 which is F12, "Heat Recovery Reference Flow Rate", defaulted to blank for autosize?
+There are three new optional heat recovery fields at the end which transition will not add.
+
+# Object Change: HeatPump:PlantLoop:EIR:Heating
+
+Insert new field A7 which is also F7, "Heat Recovery Inlet Node Name", defaulted to blank?
+Insert new field A8 which is also F8, "Heat Recovery Outlet Node Name", defaulted to blank?
+Insert new field N3 which is F12, "Heat Recovery Reference Flow Rate", defaulted to blank for autosize?
+There are three new optional heat recovery fields at the end which transition will not add.
```

Comment:
```suggestion
# Object Change: HeatPump:PlantLoop:EIR:Cooling

Insert new field A7 which is also F7, "Heat Recovery Inlet Node Name", defaulted to blank
Insert new field A8 which is also F8, "Heat Recovery Outlet Node Name", defaulted to blank
Insert new field N3 which is F12, "Heat Recovery Reference Flow Rate", defaulted to blank for autosize
There are three new optional heat recovery fields at the end which transition will not add.

# Object Change: HeatPump:PlantLoop:EIR:Heating

Insert new field A7 which is also F7, "Heat Recovery Inlet Node Name", defaulted to blank
Insert new field A8 which is also F8, "Heat Recovery Outlet Node Name", defaulted to blank
Insert new field N3 which is F12, "Heat Recovery Reference Flow Rate", defaulted to blank for autosize
There are three new optional heat recovery fields at the end which transition will not add.
```

Reply:
If everything looks good, we can commit this suggested change and merge this in.